save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/TopBar.kt

```diff
@@ -54,6 +54,148 @@ external interface TopBarProps : PropsWithChildren {
     var userInfo: UserInfo?
 }
 
+/**
+ * The class for analyzing url address and creating links in topBar
+ */
+class TopBarUrl(href: String) {
```

Comment:
do we need such big class inside another file? maybe extract to a separate file?

Reply:
Done